types/config.d.ts

```diff
@@ -242,15 +242,15 @@ type ValueType =
   | 'relative-size'
   | 'shadow'
 export interface PluginAPI {
-  /** for registering new static utility styles */
+  // for registering new static utility styles
```

Comment:
These could also be useful to the user, or do you reckon that this is a bit redundant because the name `addUtilities` itself is self explanatory enough?

Reply:
I agree that we could write some documentation comments that are useful to users, but it seems like the comments in this file right now have not been written for that purpose, and it was not the intention for these to be visible to users. Consider comments like `Utilities` or `Responsiveness` for example. I propose that we hide everything for now and consider adding proper documentation comments in the future.

Reply:
Yep sounds good, let's merge it 💪